wolfssl/wolfcrypt/settings.h

```diff
@@ -409,6 +412,16 @@
     #endif
 #endif
 
+#ifdef WOLFSSL_EMBOS
+    #define USE_FAST_MATH
+    #define HAVE_ECC
+    #define NO_DEV_RANDOM
```

Comment:
I don't understand how a #warning would generate a seed.  You have defined NO_DEV_RANDOM which currently does:

#elif defined(NO_DEV_RANDOM)

#error "you need to write an os specific wc_GenerateSeed() here"

/*
int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
{
    return 0;
}
*/

The pull request doesn't reflect the seeding that I can tell.